.github/workflows/coderabbit-autofix.yml

```
@@ -0,0 +1,469 @@
+    runs-on: ubuntu-latest
+    timeout-minutes: 2
+    outputs:
+      should_run: ${{ steps.check.outputs.should_run }}
+      attempt_number: ${{ steps.check.outputs.attempt_number }}
+      review_body: ${{ steps.fetch-review.outputs.review_body }}
+      inline_comments: ${{ steps.fetch-review.outputs.inline_comments }}
+    steps:
+      # GUARD 1: Verify reviewer is CodeRabbit bot
+      - name: Check reviewer is CodeRabbit
+        id: check-reviewer
+        run: |
```

actionlint (1.7.10)

shellcheck reported issue in this script: SC2086:info:6:33: Double quote to prevent globbing and word splitting

(shellcheck)